piControlMain.h

```diff
@@ -70,6 +70,13 @@ typedef struct spiControlDev {
 	struct device *dev;
 	struct thermal_zone_device *thermal_zone;
 
+	// supports extension modules with RS485 based communication (eg. DIO or AIO)
+	unsigned int pibridge_rs485_supported:1;
+	// device supports RevPi gateways (only point to point communication)
+	unsigned int pibridge_ethernet_p2p_supported:1;
+	// device support extension modules with ethernet based communication (for future use)
+	unsigned int pibridge_ethernet_supported:1;
+
```

Comment:
Usually in C, such flags are given short and succinct names to minimize the amount of line-wrapping that needs to be done. That's why I suggested `revpi_gate_supported` and `pibridge_supported`. That's already quite long but one has to strike a balance between readability and shortness. ;)